cloud/terraform/terraform.py

```diff
@@ -1,12 +1,15 @@
 from ast import alias
 import os
 import json
-
+import sys
 
 class TerraformConfigurator:
     def __init__(self, cloud):
         self.cloud = cloud
 
+        # Change cwd to file path
+        os.chdir(sys.path[0])
```

Comment:
Good point, creating `main.tf` programmatically would be a good option. Otherwise, we will have to copy the `main.tf` default file to the place where `providers.tf` and `resources.tf` will be created. Maybe it's cleaner to create `main.tf` programmatically. This taking into account that we remove the `.tf` files as part of the "destroy" stage.